### PR TITLE
feat: florencev2 to support video

### PR DESCRIPTION
**Pull request by junjie-landing**
per convo, break down video frame-by-frame for now.

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+set -e # exit on error
+
+sudo apt-get update && sudo apt-get upgrade
+curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+python get-pip.py
+curl -sSL https://install.python-poetry.org | python -
+poetry run pip install --upgrade pip setuptools toml
+if [[ "$(uname)" == "Darwin" ]]; then
+	brew install zbar
+elses
```
Comment (hrnn) [inline scripts/install.sh:13]: I see the unit test is not installing `zbar` is this a typo? 

```suggestion
else
```

Comment (junjie-landing) [reply scripts/install.sh]: oops!

File: vision_agent_tools/models/florencev2.py [hunk 2]
```diff
@@ -65,20 +72,28 @@ def __init__(self):
         self._processor = AutoProcessor.from_pretrained(
             PROCESSOR_NAME, trust_remote_code=True
         )
+        # self._frame2img = get_class_from_dynamic_module(
+        #     "ixc_utils.frame2img", MODEL_NAME
+        # )
+        # self._video_transform = get_class_from_dynamic_module(
+        #     "ixc_utils.Video_transform", MODEL_NAME
+        # )
```
Comment (CamiloInx) [inline vision_agent_tools/models/florencev2.py:80]: Please remove this, we should not use this preprocessing from the IXC2 model

File: vision_agent_tools/models/florencev2.py [hunk 3]
```diff
@@ -98,8 +113,20 @@ def __call__(
...
 
-        image = image.convert("RGB")
+        # Either video or image should be provided
+        if image is None and video is None:
+            raise ValueError("Either 'image' or 'video' must be provided.")
+        if image is not None and video is not None:
+            raise ValueError("Only one of 'image' or 'video' can be provided.")
+
+        if image is not None:
+            image = self._process_image("RGB")
```
Comment (hrnn) [inline vision_agent_tools/models/florencev2.py:123]: Looks like the tests are failing due to this call

`_process_image` is defined as 
```python
    def _process_image(self, image: Image.Image) -> Image.Image:
        return image.convert("RGB")
```

so it is executing `"RGB".convert("RGB")` thus the error

```
E       AttributeError: 'str' object has no attribute 'convert'
```

**Review by hrnn — APPROVED** [review]
:shipit:

**Pull request merged**